.github/workflows/docker.yml

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 Intel Corporation
+name: Docker image
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout and build
+    - uses: actions/checkout@v3
+    - name: Build and push Docker image
+      run: |
+        make docker-build
+
+  push-dockerhub:
+    env:
+      DOCKER_REGISTRY: "docker.io/"
+      DOCKER_REPOSITORY: "badhrinathpa/"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: echo GIT_SHA_SHORT=$(git rev-parse --short HEAD) >> $GITHUB_ENV
+
+      - id: docker-login
+        uses: docker/login-action@v2.1.0
+        with:
+          registry: docker.io
+          username: ${{ secrets.DOCKER_HUB_LOGIN }}
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+
+      - name: Build and push "master-latest" Docker images
```

Comment:
Are we using this image anywhere ? Else just compiling should be good enough ?

Reply:
will comment out the docker push part. can enable later.